src/app/app.lastSearchesService.js

```diff
@@ -0,0 +1,29 @@
+export default class LastSearchesService {
+    constructor(lastSearches) {
+        this.lastSearches = lastSearches;
+    }
+
+    saveCity(city) {
+        if (city === undefined || city.trim() === '') {
+            return;
+        } else {
+            const cityAux = (city.charAt(0).toUpperCase() + city.slice(1).toLowerCase()).trim();
```

Comment:
why is this needed?

Reply:
To save and show the names of the cities with the first letter in capital, and the rest with lower case